src/Home/index.tsx

```diff
@@ -62,9 +65,18 @@ const Home: FunctionComponent = () => {
   const headerText = appIsActive
     ? t("home.bluetooth.tracing_on_header")
     : t("home.bluetooth.tracing_off_header")
-  const subheaderText = appIsActive
-    ? t("home.bluetooth.all_services_on_subheader", { applicationName })
-    : t("home.bluetooth.tracing_off_subheader")
+  const subheaderText = () => {
+    switch (appIsActive) {
```

Comment:
why the switch here? could and if-else block work?

Reply:
Good call.